README.md

```diff
@@ -83,7 +114,7 @@ Note: this will completely remove your credentials from RD Station (`update_acce
 Returns data about a specific Contact
 
 ```ruby
-client = RDStation::Client.new(access_token: 'access_token')
+client = RDStation::Client.new(access_token: 'access_token', refresh_token: 'refresh_token')
```

Comment:
Um outro ponto que me veio em mente, há algum motivo de não passarmos o refresh_token também no configure? Evitando chamar aqui também.

O que acham?

(Não é impeditivo, mais como melhoria mesmo)

Reply:
Por que o refresh token é do usuário e o client_id e client_secret são da app. Uma mesma app pode ter inúmeros refresh tokens e acess tokens (como por exemplo o `rd-pipedrive`)

Reply:
Ele não se encaixa nesse ponto. Só o `client_id` e o `client_secret` podem ser configurados na inicialização mesmo.

O `refresh_token` só pode ser obtido fazendo a autenticação OAuth via `RDStation::Authentication`. No ponto de vista de uma integração, vai existir um `refresh_token` diferente pra cada conta que usa a integração. Enquanto isso, o `client_id` e o `client_secret` identificam a integração como um todo.

Reply:
Humm entendi :v: